tests/rptest/services/openmessaging_benchmark_configs.py

```diff
@@ -201,6 +232,21 @@ def validate_metrics(metrics, validator):
         "test_duration_minutes": 15,
         "warmup_duration_minutes": 5,
     }
+
+    # Release certifying smoke test we use for perf run.
+    RELEASE_CERT_SMOKE_LOAD_625k = {
+        "name": "SmokeLoad625kReleaseCert",
+        "topics": 1,
+        "partitions_per_topic": 100,
+        "subscriptions_per_topic": 1,
+        "consumer_per_subscription": 8,
+        "producers_per_topic": 16,
+        "producer_rate": 625000,
+        "consumer_backlog_size_GB": 0,
+        "test_duration_minutes": 5,
```

Comment:
@rystsov Does it usually catch regressions in such a short window? or should we bump it up?

Reply:
It would have caught the idempotency regression if it was there. We may start with 5 min and later increase the coverage of the tests for example to exercise different cleanup policies